Multisig/UI/Transaction/TransactionDetailsViewController/TransactionDetailsViewController.swift

```diff
@@ -76,31 +79,71 @@ class TransactionDetailsViewController: LoadableViewController, UITableViewDataS
+        actionsContainerView.axis = .horizontal
+        actionsContainerView.distribution = .fillEqually
+        actionsContainerView.alignment = .fill
+        actionsContainerView.spacing = 20
+        actionsContainerView.translatesAutoresizingMaskIntoConstraints = false
+
+        rejectButton = UIButton(type: .custom)
+        rejectButton.setText("Reject", .filledError)
+        rejectButton.addTarget(self, action: #selector(didTapReject), for: .touchUpInside)
+        actionsContainerView.addArrangedSubview(rejectButton)
+
         confirmButton = UIButton(type: .custom)
         confirmButton.setText("Confirm", .filled)
         confirmButton.addTarget(self, action: #selector(didTapConfirm), for: .touchUpInside)
-        confirmButton.translatesAutoresizingMaskIntoConstraints = false
-        view.addSubview(confirmButton)
+        actionsContainerView.addArrangedSubview(confirmButton)
+
+        view.addSubview(actionsContainerView)
         NSLayoutConstraint.activate([
-            confirmButton.heightAnchor.constraint(equalToConstant: 56),
-            confirmButton.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -8),
-            confirmButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 16),
-            confirmButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -16)
+            actionsContainerView.heightAnchor.constraint(equalToConstant: 56),
+            actionsContainerView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -8),
+            actionsContainerView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 16),
+            actionsContainerView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -16)
         ])
     }
 
     override func showOnly(view: UIView) {
         super.showOnly(view: view)
-        confirmButton.isHidden = view !== tableView || !showsConfirmButton
+        actionsContainerView.isHidden = view !== tableView || !showsActionsViewContrainer
+        confirmButton.isHidden = !showConfirmButton
+        rejectButton.isHidden = !showsRejectButton
+
+        confirmButton.isEnabled = enableConfirmButton
+        rejectButton.isEnabled = enableRejectionButton
     }
 
-    private var showsConfirmButton: Bool  {
-        App.configuration.toggles.signing && tx?.txStatus == .awaitingYourConfirmation
+    private var showsActionsViewContrainer: Bool  {
+        App.configuration.toggles.signing && (showsRejectButton || showConfirmButton)
+    }
+
+    private var showsRejectButton: Bool {
+        switch self.tx?.txInfo {
+
+        case .rejection(_):
+            return false
+        default:
+            return tx?.multisigInfo != nil
+        }
+    }
+
+    private var showConfirmButton: Bool {
+         [.awaitingYourConfirmation, .awaitingConfirmations].contains(tx?.txStatus)
+    }
+
+    private var enableRejectionButton: Bool {
+        return true
```

Comment:
will this be updated in the next PR according to the specs we clarified?

Reply:
Yes